conans/client/conan_command_output.py

```diff
@@ -30,9 +30,15 @@ def profile_list(self, profiles):
     def remote_list(self, remotes, raw):
         for r in remotes:
             if raw:
-                self._output.info("%s %s %s" % (r.name, r.url, r.verify_ssl))
+                disabled_str = " True" if r.disabled else ""
+                self._output.info(
+                    "%s %s %s %s" %
+                    (r.name, r.url, r.verify_ssl, disabled_str))
```

Comment:
@lasote, I want you to be aware of this change: the **raw output** will be modified by this change, we are introducing another column to some rows.

Reply:
Thanks for the warn. I would say we can take the risk and just in case of someone reporting revert and annotate for 2.0.

Reply:
This will only produce a different output (modulo some extra blank space), if they are using this new "disabling remote" feature, otherwise the output will be the same and a parser that ignores or strips blanks will be fine.